### DISCUSSION TITLE
升级2.2.29后，table中cell的默认padding消失了，要如何处理才能恢复原来的效果呢？

### DISCUSSION THREAD

**Discussion by lk101** [element-plus/element-plus#11432] [Q&A] [answered]
vue版本3.2.47，element-plus版本2.2.29，Windows10，Chrome最新版本

我注意到是因为#10708这个PR导致的此样式变化，并且其中支持了default的padding设置，但由于之前并没有做过任何相应设置，对于如何添加$table-cell-padding变量，我并没有相关经验，请各位有以教我。

**Comment by tolking** [comment, marked as answer]
抱歉审查的时候没有注意到该问题

问题产生的原因是：size 的默认值是 `''`，这里错误的将其作为 `'default'` 使用引起的

临时解决办法：增加配置 `<el-table size="default">`

在 #10708 中应该保留 .cell 的 padding 样式。如果你对修复它感兴趣，可以提 PR 进行修复。

**Reply by tolking** [reply]
Fixed in #11456
